docs/WebEditorBasics/WebPlayMode.md

```diff
@@ -0,0 +1,53 @@
+---
+id: WebPlayMode
+title: Web Play Mode
+---
+
+Don't always have access to a device? Waiting for a battery to recharge? No problem!
+
+You can continue to build with Enklu Cloud's blazing fast iteration times by using Web Play Mode to test your experiences.
+
+// TODO: Liam's awesome gif here.
+
+## What is Web Play Mode?
+
+Web Play Mode is where your experience comes to life, without the extra weight of the editor tools. 
+On device, this is the default mode that experiences load into. From Enklu Cloud, you can switch to play mode at any time while developing an experience.
+
+This can be useful for getting a first person view of the experience and testing any potential scripting that occurs at the start of a session.
+While in Web Play Mode, edits made by you or other collaborators will update live, similar to being in HoloLens.
+
+> Note: Interactions are limited to mouse clicks. There is currently no emulation for device specific input like hand tracking.
+
+## Using Web Play Mode
+
+To start viewing your experience in Play Mode, click on the Play icon above the canvas. 
+A new window will open and begin loading the experience. The existing editor window can still be used to develop side by side with Play Mode.
+
+![Play Button](/img/product/editor/ControlBar.png)
```

Comment:
Not a huge deal, but I feel like a more zoomed in image would be better here.  Zoomed in as in CTRL + in chrome not like cutting anything off.  The play button is just so small especially if you don't know what you're looking for.